src/css-module-dts-loader/loader.ts

```diff
@@ -33,6 +33,9 @@ const mTimeMap = new Map<string, Date>();
 
 function generateDTSFile(filePath: string): Promise<void> {
 	return Promise.resolve().then(() => {
+		if (!/src[\\\/]/.test(filePath)) {
```

Comment:
should this be configarable from somewhere?

Reply:
I added the option to pass a regex or string pattern via the loader options.